contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -80,11 +80,11 @@ In this case, only the `root` component can be accessed in Vue while the rest ar
 <b>Step 4:</b> Pass the message to the HTML file using double curly brackets.
 ```HTML
   <div id="root">
-    <h2>{\{message}\{</h2>
+    <h2>{\{message}\}</h2>
```

Comment:
A little troublesome, but you could workaround all these occurences temporarily by using copying the html output and replacing `{\{` with `{<span>{</span>`. Might be something for another PR if there's too much though

![image](https://user-images.githubusercontent.com/3306138/74739793-8f3d0600-5294-11ea-9c1f-526b304dec0b.png)

Reply:
On second thought, there's an ongoing fix in https://github.com/MarkBind/markbind/issues/762, might be better to do it in a future PR if it needs to be changed again